src/modules/offline-page/index.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-import { cacheName as documentCache } from '../document-caching/constants';
-import { cacheName as assetCache } from '../asset-caching/constants';
+import { cacheName as AMP_PUBLISHER_CACHE_NAME } from '../document-caching/constants';
```

Comment:
Nice to avoid the `as` statements during import by naming the `cacheName` this in the first place. Helps with finding usages.